docs/Getting Started/Getting-Started-From-Scratch.md

```diff
@@ -1,100 +1,1126 @@
+    
+    
 4. Create a `genesis.json` file see example [here](../genesis). The `alloc` field should be pre-populated with the account you generated at previous step
-5. Generate node key `bootnode --genkey=nodekey` and copy it into datadir
-6. Execute `bootnode --nodekey=new-node-1/nodekey --writeaddress` and take note of the displayed output. This is the enode id of the new node
+    ```
+    $ vim genesis.json
+    ... paste below with account '679fed8f4f3ea421689136b25073c6da7973418f' created at previous step
+    {
+      "alloc": {
+        "0x679fed8f4f3ea421689136b25073c6da7973418f": {
+          "balance": "1000000000000000000000000000"
+        }
+      },
+     "coinbase": "0x0000000000000000000000000000000000000000",
+     "config": {
+       "homesteadBlock": 0,
+       "byzantiumBlock": 0,
+       "chainId": 10,
+       "eip150Block": 0,
+       "eip155Block": 0,
+       "eip150Hash": "0x0000000000000000000000000000000000000000000000000000000000000000",
+       "eip158Block": 0,
+       "isQuorum": true
+      },
+     "difficulty": "0x0",
+     "extraData": "0x0000000000000000000000000000000000000000000000000000000000000000",
+     "gasLimit": "0xE0000000",
+     "mixhash": "0x00000000000000000000000000000000000000647572616c65787365646c6578",
+     "nonce": "0x0",
+     "parentHash": "0x0000000000000000000000000000000000000000000000000000000000000000",
+     "timestamp": "0x00"
+    }   
+    ```
+5. Generate node key and copy it into datadir
+    ```
+    $ bootnode --genkey=nodekey
+    $ cp nodekey new-node-1/
+    ```
+6. Execute below command to display enode id of the new node
+    ```
+    $ bootnode --nodekey=new-node-1/nodekey --writeaddress
+    '70399c3d1654c959a02b73acbdd4770109e39573a27a9b52bd391e5f79b91a42d8f2b9e982959402a97d2cbcb5656d778ba8661ec97909abc72e7bb04392ebd8'
+    ```
 7. Create a file called `static-nodes.json` and edit it to match this [example](../permissioned-nodes). Your file should contain a single line for your node with your enode's id and the ports you are going to use for devp2p and raft. Ensure that this file is in your nodes data directory
-8. Initialize new node with `geth --datadir new-node-1 init genesis.json`
-9. Start your node and send into background with `PRIVATE_CONFIG=ignore nohup geth --datadir new-node-1 --nodiscover --verbosity 5 --networkid 31337 --raft --raftport 50000 --rpc --rpcaddr 0.0.0.0 --rpcport 22000 --rpcapi admin,db,eth,debug,miner,net,shh,txpool,personal,web3,quorum,raft --emitcheckpoints --port 21000 2>>node.log &`
+    ```
+    $ vim static-nodes.json
+    .... paste below lines with enode generated in previous step, port 21000;IP 127.0.0.1 and raft port set as 50000
+    [
+      "enode://70399c3d1654c959a02b73acbdd4770109e39573a27a9b52bd391e5f79b91a42d8f2b9e982959402a97d2cbcb5656d778ba8661ec97909abc72e7bb04392ebd8@127.0.0.1:21000?discport=0&raftport=50000"
+    ] 
+    ```
```

Comment:
add the copy command to copy the static-nodes.json to new-node-1
`cp static-nodes.json new-node-1`